app/controllers/articles_controller.rb

```diff
@@ -33,6 +40,12 @@ def create
     respond_to do |format|
       if @article.save
         format.html { redirect_to @article, notice: 'Article was successfully created.' }
+        format.js do
+          render js: <<-endjs
+            window.location = '#{articles_path}';
+          endjs
```

Comment:
The increased coherency here is appreciated, but it's more conventional to keep your js view in a `create.js.erb` file in your `/views` directory. An argument could be made for this approach, tho!